src/front-end/typescript/lib/pages/opportunity/code-with-us/edit/tab/addenda.tsx

```diff
@@ -43,8 +43,9 @@ const update: component_.page.Update<State, InnerMsg, Route> = ({
   switch (msg.tag) {
     case "onInitResponse": {
       const opportunity = msg.value[0];
+      const existingAddenda = opportunity.addenda;
       const [addendaState, addendaCmds] = Addenda.init({
-        existingAddenda: [],
+        existingAddenda: existingAddenda,
```

Comment:
FYI you can simply write `existingAddenda,` here instead of the `existingAddenda: existingAddenda`.